DEVICES.md

```diff
@@ -616,6 +616,7 @@ of device.
 - Unbranded 1CH dimmer module
 - Unbranded dual dimmer module
 - A60 1800-2700K RGBWW light
+- Arlec Grid Connect 6w RGB+CCT Downlight(Under Generic RGBCW/RGBWW lightbulb)
```

Comment:
Don't add entries for devices that share a generic config. This file is too big already to be practical.